### DISCUSSION TITLE
Last evaluation or best evaluation?

### DISCUSSION THREAD

**Discussion by fersebas** [PaddlePaddle/PaddleSpatial#147] [BaiduKDDCup2022] [answered]
Hi! The leaderboard show the last evaluation instead of the best evaluation....
Why?

Thank you

**Comment by fersebas** [comment]
![image](https://user-images.githubusercontent.com/49498032/172057162-2e1a86d4-49b9-4509-a9c7-0fe0d8e230a1.png)
Team datateam-UCM

**Comment by ramber1836** [comment, marked as answer]
The best result is shown on the Leaderboard. 
In your case, your submission with the best score was re-evaluated, but an error occurred, so your best result does not show up. You could fix the error and resubmit your code to obtain your best score.
